other/copy-namespace-labels/.chainsaw-test/patchresource.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: my-deployment-1
+  namespace: within-ns
+  labels:
+    app: good-app
+    owner: "any-corp"
+    env: dev
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: good-app
+  template:
+    metadata:
+      labels:
+        app: good-app
+    spec:
+      containers:
+      - name: good-app-deploy
+        image: busybox:1.35
+---
+#Deployment with `kubernetes.io/metadata.name` label
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: my-deployment-2
+  namespace: within-ns
+  labels:
+    owner: "any-corp"
+    env: dev
```

Comment:
You're missing the `app` label from the source Deployment. You must assert that it is present as well.